modules/bar/window_title/index.ts

```diff
@@ -132,28 +133,55 @@ const filterTitle = (windowtitle: ActiveClient) => {
     };
 };
 
+const defaultTitle = (client: ActiveClient) => {
+    let title = client.title;
+    // If the title is empty or only filled with spaces, fallback to the class name
+    if (title.length === 0 || title.match(/^ *$/)) {
+        title = client.class;
+    }
+    return title;
```

Comment:
```suggestion
    let title = client.title;
    // If the title is empty or only filled with spaces, fallback to the class name
    if (title.length === 0 || title.match(/^ *$/)) {
        title = client.class;
    }
    return title;
```
We return class by default for windows. I don't necessarily dislike having the title as well but if we're going to display a title we should make that an option instead.

Everything else looks great :+1: